org.caveman250.gtkeddit.json

```diff
@@ -0,0 +1,163 @@
+{
+  "app-id": "org.caveman250.gtkeddit",
+  "version": "0.1",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "40",
+  "sdk": "org.gnome.Sdk",
+  "command": "gtkeddit",
+  "finish-args": ["--share=network", "--socket=wayland", "--socket=fallback-x11", "--socket=session-bus", "--filesystem=home" ],
```

Comment:
Why does it need full access to session-bus and home?

Reply:
removed the home access, wasn't needed.

The session bus was a fix for a crash on launch during gtk initialization that has me a bit puzzled.

> Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

Reply:
You can check which D-Bus names are needed by your app by running it with `flatpak run --log-session-bus`.

Reply:
thanks! Thats fixed, removed the need for session-bus